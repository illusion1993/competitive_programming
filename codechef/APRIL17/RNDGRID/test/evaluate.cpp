#include<bits/stdc++.h>
using namespace std;

int t, m, n, prob;
string path;
char board[1000][1000];

int main(int argc, char const *argv[])
{
    ifstream tin ("testcases.in");
    ifstream ain ("brute.out");
    ifstream bin ("opt.out");

    ofstream cout ("evaluate.out");

    int c = 0, a, b;
    tin >> t;
    while(t--) {
        tin >> m >> n >> path;
        for (int i = 0; i < n; i++) for (int j = 0; j < n; j++) tin >> board[i][j];

        ain >> a;
        bin >> b;
        if (a != b) {
            c++;
            cout << m << " " << n << endl << path << endl;
            for (int i = 0; i < n; i++) {
                for (int j = 0; j < n; j++) cout << board[i][j];
                cout << endl;
            }
        }
    }
    cout << "\nTotal: " << c;

    return 0;
}
