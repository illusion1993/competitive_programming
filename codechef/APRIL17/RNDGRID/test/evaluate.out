10
4 6
DULD
...#..
#.....
#.#...
......
..#.#.
###...
4 6
RUDL
.....#
#.....
..#.#.
#.#...
.#....
...#..
4 6
LDRU
.#....
......
..#..#
...#..
.#.#..
......
4 6
DULD
#....#
....#.
..#.#.
..#...
......
......
4 6
ULRU
#...##
#..#..
......
.#....
##....
....#.
4 6
LDRD
......
.##.#.
.#...#
......
..#...
.#.##.
4 6
LDUU
#.....
....##
......
......
..#..#
......
4 6
URDU
...#..
......
......
.#...#
..#...
.....#
4 6
LLDD
...#..
......
.#.#..
.....#
..#...
#..#.#
4 6
LRDD
..#...
#.....
#.#...
......
..#...
.....#